src/rendering/RenderUtil.cc

```diff
@@ -1824,12 +2018,14 @@ void RenderUtilPrivate::RemoveRenderingEntities(
       });
 }
 
+static std::mutex mutexInit;
```

Comment:
Should this be a part of the `RenderUtil` class instead of being declared globally?